src/controller/cve.controller/index.js

```diff
@@ -183,7 +159,7 @@ router.post('/cve/:id',
     }
   }
   #swagger.responses[200] = {
-    description: 'The requested CVE record is returned.',
+    description: 'The requested CVE record is returned',
     content: {
       "application/json": {
         schema: { $ref: '/schemas/cve/create-cve-record-response.json' }
```

Comment:
Response JSON should match this example
{
    "message": "CVE-1999-0005 record was successfully created.",
    "created": {
        "dataType": "CVE_RECORD",
        "dataVersion": "5.0",
        "cveMetadata": {
            "cveId": "CVE-1999-0005",
            "assignerOrgId": "ee1fa4c6-8d34-4353-ad97-194c1b986b8b",
            "state": "PUBLISHED",
            "assignerShortName": "exampleCNA",
            "requesterUserId": "adfb94fe-a90a-42a5-8ce4-12aa1f897cde",
            "dateReserved": "2022-04-04T02:28:12.844Z",
            "datePublished": "2022-04-04T02:29:23.283Z"
        },
        "containers": {
            "cna": {
                "x_a": {
                    "bb": "c"
                },
                "affected": [
                    {
                        "product": "p",
                        "vendor": "v",
                        "versions": [
                            {
                                "version": "1.2",
                                "status": "affected"
                            }
                        ]
                    }
                ],
                "descriptions": [
                    {
                        "lang": "en",
                        "value": "v p 1.2 is insecure."
                    }
                ],
                "problemTypes": [
                    {
                        "descriptions": [
                            {
                                "description": "insecurity",
                                "lang": "en"
                            }
                        ]
                    }
                ],
                "providerMetadata": {
                    "orgId": "ee1fa4c6-8d34-4353-ad97-194c1b986b8b",
                    "shortName": "exampleCNA"
                },
                "references": [
                    {
                        "url": "https://example.com"
                    }
                ]
            }
        }
    }
}